### PR TITLE
Add PingCAP use case

### PR DESCRIPTION
**Pull request by dcalvin**
Signed-off-by: Calvin Weng <wenghao@pingcap.com>
This PR updated the main page and About page with details and files for PingCAP use case of gRPC.

### CODE REVIEW

**Comment by srini100** [comment]
@thisisnotapril, would you like to review this?

File: content/about.md [hunk 1]
```diff
@@ -111,7 +111,17 @@ Check out what people are saying below:
...
-</div>    
+</div>
+
+<div class="testimonialsection">
+    <div class="testimonialimage"><a href="http://www.pingcap.com">
+        <img src="../img/pingcap5.png" /></a></div>
+        <div>
+            <div class="testimonialquote">“ </div>
+            At PingCAP, we use gRPC in almost every project that involves interactions or communications through network. As has been proven, gRPC has well met our demands in terms of performance and usability. We also provide gRPC library for Rust, which is the first production-ready gRPC implementation in the Rust community.
+        </div>
```
Comment (srini100) [inline content/about.md]: The claim "first" is hard to prove. grpc.io doesn't want to endorse such claims. Can you please reword it to not include "first"

Comment (dcalvin) [reply content/about.md]: Thanks for the comment. What about "which is among the first few production-ready gRPC implementations in the Rust community"

Comment (chalin) [reply content/about.md]: Thanks for the suggestion @dcalvin. I'll let @thisisnotapril comment on it.

How about simply:
> We also provide a production-ready gRPC library for Rust.

@thisisnotapril WDYT?

Comment (dcalvin) [reply content/about.md]: @chalin Your suggestion sounds good to me! We are good with either your simple version or mine.  I will update when @thisisnotapril comments.

File: content/about.md [hunk 2]
```diff
@@ -111,7 +111,17 @@ Check out what people are saying below:
...
 
-</div>    
+</div>
+
+<div class="testimonialsection">
+    <div class="testimonialimage"><a href="http://www.pingcap.com">
+        <img src="../img/pingcap5.png" /></a></div>
+        <div>
+            <div class="testimonialquote">“ </div>
+            At PingCAP, we use gRPC in almost every project that involves interactions or communications through network. As has been proven, gRPC has well met our demands in terms of performance and usability. We also provide gRPC library for Rust, which is the first production-ready gRPC implementation in the Rust community.
```
Comment (dcalvin) [inline content/about.md]:
```suggestion
            At PingCAP, we use gRPC in almost every project that involves interactions or communications through network. As has been proven, gRPC has well met our demands in terms of performance and usability. We also provide a production-ready gRPC library for Rust.
```

File: content/about.md (hunk 1 above)
Comment (dcalvin) [reply content/about.md]: @chalin Updated as you suggested.

**Comment by thisisnotapril** [comment]
lgtm!
